pkg/controller/infinispan/infinispan_controller.go

```diff
@@ -835,10 +832,10 @@ func (r *ReconcileInfinispan) statefulSetForInfinispan(m *infinispanv1.Infinispa
 			dep.Spec.Template.Spec.InitContainers = []corev1.Container{{
 				Image:   initContainerImage,
 				Name:    "chmod-pv",
-				Command: []string{"sh", "-c", "chmod -R g+w /opt/infinispan/server/data"},
+				Command: []string{"sh", "-c", "chmod -R g+w" + DataMountPath},
```

Comment:
@ryanemerson you are missing the reason why CI fails
```suggestion
				Command: []string{"sh", "-c", fmt.Sprintf("chmod -R g+w %s", DataMountPath)},
```